### ISSUE TITLE
Feature request: some visibility into passing tests call sequences

### ISSUE THREAD

**Issue by moodysalem** [crytic/echidna#534]
In writing echidna tests, it's hard to know whether your properties are passing because the code is good, or if you are not getting coverage of the bad test cases, e.g. need to increase test gas limit

It would be great to be able to pass a parameter, e.g. `--samples 100` and get 100 random examples of calls printed along with the test results for successful cases to have some visibility into the coverage that echidna is getting

**Comment by ggrieco-tob** [comment]
We are already working in coverage support and it works!. Check PR #516. To enable printing of covered code, you should use `format: text` and `coverage: true` in your .yaml file. The only issue is that we are blocked by this issue https://github.com/crytic/crytic-compile/issues/112. The coverage won't print correctly if your source code uses more than one file (but you can flatten the contracts using `slither --flatten`).

**Comment by moodysalem** [comment]
This is great! Even with code coverage though, the inputs and the call order matters. Does this feature only print line level coverage? Or also show some information about the inputs?

**Comment by ggrieco-tob** [comment]
So far this code only prints covered lines. You could, in teory, take a look to the `corpusDir` directory, but there is no easy way to check which input correspond to each line. We could easily print if the information per line, at least we want to signal the user if the line is covered in a revert or not but perhaps we could add more. Please let us know what you need.

**Comment by ggrieco-tob** [comment]
Priting coverage already works fine. I will close this, open another issue if you need some more specific feature.
